.eslintrc.json

```diff
@@ -34,7 +34,8 @@
     "default-case": 0,
     "import/no-extraneous-dependencies": 0,
     "no-cond-assign": 0,
-    "array-callback-return": 0,
-    "function-paren-newline": 0
+    "import/no-unresolved": 0,
```

Comment:
Hound CI can't parse this rule. Also, it depends on current installation settings so strictly following it won't be helpful.